organisaatio-ui/src/components/Sivut/LomakeSivu/UusiToimija/PerustietoLomake/PerustietoLomake.tsx

```diff
@@ -46,14 +53,15 @@ export default function PerustietoLomake(props: OrganisaatioProps) {
     const [nimenmuutosModaaliAuki, setNimenmuutosModaaliAuki] = useState<boolean>(false);
     const [onYunnus, setOnYtunnus] = useState<boolean>(true);
     const { kuntaKoodisto } = useContext(KoodistoContext);
-    const kaikkiKunnat = kuntaKoodisto.koodit().map((k: any) => ({
+    const kaikkiKunnat = kuntaKoodisto.koodit().map((k) => ({
         value: k.uri,
-        label: k.nimi[language] || k.nimi['fi'] || k.nimi['sv'] || k.nimi['en'],
+        label: k.nimi[language] || k.nimi['fi'] || k.nimi['sv'] || k.nimi['en'] || '',
     }));
     const [nimi, setNimi] = useState(tyhjaNimi);
 
     const handleNimiTallennus = () => {
         const nimet = { nimi: Object.assign({}, nimi), alkuPvm: new Date().toISOString().split('T')[0] };
+        console.log(nimet);
```

Comment:
maybe redundant to output these?

Reply:
I think we need a separate issue to remove all development log outputs